yoda.spec

```diff
@@ -11,7 +11,7 @@ Requires: boost python cython
 %prep
 %setup -q -n %{n}/%{realversion}
 
-./configure --prefix=%i --with-boost=${BOOST_ROOT} CXX="$(which %cms_cxx)" CXXFLAGS="%cms_cxxflags"
+./configure --prefix=%i CXX="$(which %cms_cxx)" CXXFLAGS="%cms_cxxflags" --enable-root
```

Comment:
Hopefully it doesn't pick ROOT from system standard location is one is available. Is there a way to pass a ROOT directory, e.g. `--with-root=<..>`?